onelogin_rules/onelogin_active_login_activity.py

```diff
@@ -1,6 +1,12 @@
 import time
+
 from panther_base_helpers import is_ip_in_network
-from panther_oss_helpers import get_string_set, put_string_set, set_key_expiration, add_to_string_set  # pylint: disable=line-too-long
+from panther_oss_helpers import (  # pylint: disable=line-too-long
```

Comment:
Now that the linter breaks this out into a multi line import, we don't need the `pylint` exception I think.
```suggestion
from panther_oss_helpers import (
```